mod-source-record-manager-server/src/main/java/org/folio/dao/JournalRecordDaoImpl.java

```diff
@@ -134,7 +134,7 @@ public class JournalRecordDaoImpl implements JournalRecordDao {
   private static final String SELECT_BY_JOB_EXECUTION_ID_QUERY = "SELECT * FROM %s.%s WHERE job_execution_id = $1";
   private static final String ORDER_BY_PATTERN = " ORDER BY %s %s";
   private static final String DELETE_BY_JOB_EXECUTION_ID_QUERY = "DELETE FROM %s.%s WHERE job_execution_id = $1";
-  private static final String GET_JOB_LOG_ENTRIES_BY_JOB_EXECUTION_ID_QUERY = "SELECT * FROM get_job_log_entries('%s', '%s', '%s', %s, %s, %b)";
```

Comment:
no need to change this for this PR due to time crunch. I would like to see more parameterized queries rather than string substitution approach that we currently use.

Reply:
I'll take a note